src/MagnumPlugins/ObjImporter/ObjImporter.cpp

```diff
@@ -28,81 +28,338 @@
 #include <fstream>
 #include <limits>
 #include <sstream>
+#include <tuple>
+#include <stdlib.h>
 #include <unordered_map>
 #include <Corrade/Containers/ArrayView.h>
 #include <Corrade/Utility/String.h>
+#include <Corrade/Utility/Directory.h>
 
 #include "Magnum/Mesh.h"
 #include "Magnum/MeshTools/CombineIndexedArrays.h"
 #include "Magnum/MeshTools/Duplicate.h"
+#include "Magnum/Math/Vector3.h"
 #include "Magnum/Math/Color.h"
 #include "Magnum/Trade/MeshData3D.h"
+#include "Magnum/Trade/MeshObjectData3D.h"
+#include "Magnum/Trade/ImageData.h"
+#include "Magnum/Trade/PhongMaterialData.h"
+
+#include "MagnumPlugins/TgaImporter/TgaImporter.h"
+
+using namespace Corrade::Containers;
```

Comment:
It's not used on *that* many places ;)

My reasoning: if a namespace is annoyingly long to type, it indicates a problem with the API naming -- it should be shortened. Using `using namespace` would somewhat hide that problem, but not solve it, and also makes the code a bit more opaque for people that don't know what belongs to the `used` namespace. I would say that in this case the namespace is still bearable ;)